packages/web-components/src/components/leadspace/leadspace.scss

```
@@ -32,6 +32,11 @@
       opacity: 1;
     }
   }
+
+  ::slotted(#{$dds-prefix}-background-media) {
+    position: absolute;
+    inset: 0;
+  }
```

```suggestion
  ::slotted(#{$dds-prefix}-background-media) {
    position: absolute;
    @include carbon--breakpoint(md) {
      inset: 0;
    }
  }
```

Following up on @DragosRistici's comment, modifying this rule like this should make the image display on the bottom as expected.